content/id/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Volume Snapshot Class
```

Comment:
```suggestion
title: VolumeSnapshotClass
```
(because VolumeSnapshotClass is the unlocalizable resource name)

Reply:
fixed in 97c5357